src/pages/Inventory.page.js

```diff
@@ -9,7 +10,60 @@ export class InventoryPage extends BaseSwagLabPage {
 
     get addItemToCartBtns() { return this.page.locator('[id^="add-to-cart"]'); }
 
+    filterSelectLocator = this.page.locator('.select_container') 
+    
     async addItemToCartById(id) {
         await this.addItemToCartBtns.nth(id).click();
     }
+
+    async clickSorting(text){
```

Comment:
this method not only clicks but also selects a sorting type => better to rename it